### PR TITLE
added email checker

### PR DESCRIPTION
**Pull request by alishechka**
please check merci

### CODE REVIEW

File: src/main/java/com/zaichick/user/entrypoint/CreateUserVerify.java [hunk 1]
```diff
@@ -0,0 +1,16 @@
+package com.zaichick.user.entrypoint;
+import com.zaichick.user.dao.UserDao;
+import com.zaichick.user.domain.User;
+import java.util.List;
+
+public class CreateUserVerify {
+    private UserDao userDao = new UserDao();
+
+
```
Comment (rustam-drake) [inline src/main/java/com/zaichick/user/entrypoint/CreateUserVerify.java]: extra space

File: src/main/java/com/zaichick/user/entrypoint/CreateUserVerify.java [hunk 2]
```diff
@@ -0,0 +1,16 @@
+package com.zaichick.user.entrypoint;
```
Comment (rustam-drake) [inline src/main/java/com/zaichick/user/entrypoint/CreateUserVerify.java]: leave space after package
also move to "service" package "com.zaichick.user.validator"

File: src/main/java/com/zaichick/user/entrypoint/CreateUser.java [hunk 3]
```diff
@@ -14,27 +14,39 @@
...
+            createUserVerify.verifyByEmail(email);
+        } catch (Exception e){
             AwsProxyResponse awsProxyResponse = new AwsProxyResponse();
             awsProxyResponse.setStatusCode(400);
             awsProxyResponse.setBody("Duplicate email");
 
             return awsProxyResponse;
         }
 
+//        List<User> users = userDao.findUserByEmailAddress(email);
```
Comment (rustam-drake) [inline src/main/java/com/zaichick/user/entrypoint/CreateUser.java]: remove commented lines

Comment (alishechka) [reply src/main/java/com/zaichick/user/entrypoint/CreateUser.java]: done

**Pull request merged**